modern/src/map/GeofenceMap.js

```diff
@@ -72,9 +81,10 @@ const GeofenceMap = () => {
   }, []);
 
   useEffect(() => {
+    let circleFence = geofences.filter((item => item.area.indexOf('CIRCLE') > -1)).map(circleToPolygon);
     map.getSource(id).setData({
       type: 'FeatureCollection',
-      features: geofences.map(item => [item.name, reverseCoordinates(wellknown(item.area))]).filter(([, geometry]) => !!geometry).map(([name, geometry]) => ({
```

Comment:
Should we implement this circle logic in `wellknown` instead? Where is that defined?

Reply:
"wellknown" is a Node module that creates geojson object from text, and as I understand it, it doesn't handle circles, so I don't think we can use it.

Reply:
In that case we should probably move the whole logic outside of this UI component.

Reply:
Is this ok; 
new function createGeofenceFeatures in map/mapUtil.js, which will create features for all geometry types.

Reply:
I think you forgot to push the changes.

Reply:
Here it is, actually new function name is geofenceToFeature.